src/SellerLabs/Beakers/Hashing/HmacHasher.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * Copyright 2016, SellerLabs
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * This file is part of the SellerLabs package
+ */
+
+namespace SellerLabs\Beakers\Hashing;
+
+/**
+ * Class HmacHasher.
```

Comment:
All the detail on Entity makes me wonder whether there's some quick description of the use case for this class that might be good here.